web/app.fluidity.money/app/styles/dashboard.css

```diff
@@ -182,6 +182,7 @@ main > footer > section {
   overflow: hidden;
   font-size: 15px;
   box-shadow: 0 0 8px 2px #f3b8d8;
+  z-index: 1;
```

Comment:
z-indexes ?

Reply:
Yeah not sure how else to fix this issue as changing the order of the button doesn't make it layer above the graph, so here I use z-index of 1 which is then removed when the mobile modal is opened... @joshocalico do you have any other suggestions?